src/components/Modal.js

```diff
@@ -0,0 +1,28 @@
+import React, { useState } from 'react';
+import '../styles/Modal.scss';
+
+export default function Modal(props) {
+  const {
+    show,
+    onClose,
+  } = props;
```

Comment:
You can do this instead

```suggestion
export default function Modal({    show,    onClose  } ) {

```